dynalab_cli/init.py

```diff
@@ -211,13 +213,31 @@ def initialize_field(self, key, value):
             raise NotImplementedError(f"{key} not supported in setup_config")
 
     def initialize_task(self, key, value):
-        tasks = get_tasks()
-        while value not in tasks:
+        tasks, task_codes = get_tasks()
+        while value not in task_codes:
             message = (
                 f"Please choose a valid task name from one of "
-                f"[{', '.join(tasks)}]: "
+                f"[{', '.join(task_codes)}]: "
             )
             value = input(message)
+
+        task_io = [task for task in tasks if task["task_code"] == value][0][
+            "annotation_config_json"
+        ]
+        task_io = json.loads(task_io)
+
+        task_info = {"annotation_config_json": task_io, "task": value}
```

Comment:
We could call this just "annotation_config" here, because it is already loaded from a json into a python dict